app/helpers/react_on_rails_helper.rb

```diff
@@ -225,30 +226,36 @@ def server_render_js(js_expression, options = {})
 
   private
 
+  def json_safe_and_pretty(something)
+    if defined?(Rails) && Rails.env.development?
```

Comment:
Yes, `defined?(Rails) &&` should be removed.   This is artifact from where it was copy-pasted from, sorry.